DQMOffline/Trigger/src/HLTMuonMatchAndPlot.cc

```diff
@@ -484,6 +492,35 @@ void HLTMuonMatchAndPlot::analyze(Handle<MuonCollection>   & allMuons,
     }
   }
 
+  string nonSameSignPath = hltPath_;
+  bool ssPath=false;
+  if ( nonSameSignPath.rfind("_SameSign") < nonSameSignPath.length()){
+    ssPath = true;
+    nonSameSignPath = boost::replace_all_copy<string>(nonSameSignPath, "_SameSign","");
+    nonSameSignPath = nonSameSignPath.substr(0, nonSameSignPath.rfind("_v")+2);
+  }
+  bool passTriggerSS = false;
+  if (ssPath){
+    for ( unsigned int hltIndex = 0; hltIndex < numTriggers; ++hltIndex){
+      passTriggerSS = passTriggerSS ||  (trigNames.triggerName(hltIndex).find(nonSameSignPath) != std::string::npos && triggerResults->wasrun(hltIndex) && triggerResults->accept(hltIndex));
+    }
+
+    if (ssPath && targetMuons.size() > 1 && passTriggerSS){
+      if (targetMuons.at(0).charge() * targetMuons.at(1).charge() > 0){
```

Comment:
at(1) and at(0) can better be [1] and [0] since you have already checked the length of the vector.